tests/travis/install-semantic-forms-select.sh

```diff
@@ -57,7 +57,11 @@ function updateConfiguration {
 		echo '$wgLanguageCode = "'$SITELANG'";' >> LocalSettings.php
 	fi
 
-	echo 'wfLoadExtension( "SemanticForms" );' >> LocalSettings.php
+	if ( version_compare( $GLOBALS['wgVersion'], '1.28c', '>' ) ) {
```

Comment:
You carn't otherwise it will use extension.json which is broken for mw 1.27 in semanticforms.

PageForms should fix it so this is a temp solution for now.